src/sass/misc/_blog.scss

```diff
@@ -216,6 +219,14 @@
         }
     }
 
+    .social-share__icon {
+        width: 50px;
+    }
+
+    img {
+        width: 50px;
+    }
```

Comment:
I wouldn't do this as there may come a point where a post's images could be affected by this inadvertantly. The class you specified should be sufficient.